lib/tools/apk-signing.js

```diff
@@ -47,11 +47,14 @@ apkSigningMethods.executeApksigner = async function (args = []) {
   const apkSigner = await getApksignerForOs(this);
   const originalFolder = path.dirname(apkSigner);
   const getApksignerOutput = async (apksignerPath) => {
-    // Use cwd to workaround https://github.com/nodejs/node-v0.x-archive/issues/25895
-    const {stdout, stderr} = await exec(
-      util.isSubPath(apksignerPath, originalFolder) ? path.basename(apksignerPath) : apksignerPath, args, {
-        cwd: originalFolder
-      });
+    let binaryPath = apksignerPath;
+    if (system.isWindows() && util.isSubPath(binaryPath, originalFolder)) {
```

Comment:
I don't understand how the second part of this would ever be anything but true.

Reply:
easy - when `apksignerPath !== apkSigner.`This happens when `getApksignerOutput` is called with patched apksigner script, which is stored in temporary folder

Reply:
Ah! Makes sense.